src/to-be-visible.js

```diff
@@ -1,33 +1,63 @@
 import {checkHtmlElement} from './utils'
 
-function isStyleVisible(element) {
+function getElementVisibilityStyle(element) {
+  if (!element) return 'visible'
```

Comment:
This is a bit odd. If the `element` is falsy it should be considered "visible"?

Reply:
Hmmm, good point. This clarifies your point about bailing out quickly. I'll make the change.

Reply:
Yeah, for `visibility` you don't need to walk up since it the **computed** `visbility` of parents doesn't matter. only the **computed** `visibility` of the element asserted on matters.

This was relevant in earlier versions of JSOM where `visibility` wasn't inherited.